docs/Formatting.md

```diff
@@ -412,6 +412,27 @@ If `xml.format.joinContentLines` is set to `true`, the above document becomes:
   ```
 
 ***
+
+### files.trimTrailingWhitespace
+
+  Set to `true` to trim trailing whitespace when saving the file.  Defaults to `false`
```

Comment:
when saving the file --> when formatting the file

VS Code should handle trimming the trailing whitespace in the file when it's saved on its own.